app/ts/utils/constants.ts

```diff
@@ -13,6 +14,15 @@ export const REDEEM_STAKE_ADDRESS = '0x9ac7B28A7e684d1b2776d6b9045E8f9150F58401'
 export const AUDIT_FUNDS_ADDRESS = '0x73961558c6d1E8C5df845975d4D49dcA3db18887'
 export const REPUTATION_V1_TOKEN_ADDRESS = '0x1985365e9f78359a9B6AD760e32412f4a445E862'
 export const DISPUTE_CROWDSOURCER_FACTORY_ADDRESS = '0xd69769FA07c710c7b5b4c7F7f9189dbFd325976d'
+export const AUGUR_SHARE_TOKEN = '0x9e4799ff2023819b1272eee430eadf510eDF85f0'
+
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+export const UNIV4_POSITION_MANAGER = '0xbD216513d74C8cf14cf4747E6AaA6420FF64ee9e'
+export const ONE_YEAR_IN_SECONDS = 31536000n
+export const AUGUR_SHARE_DECIMALS = 15n
+export const DAI_LOGO = 'img/dai-logo.svg'
+export const YES_LOGO = 'img/yes-logo.svg'
+export const NO_LOGO = 'img/no-logo.svg'
```

Comment:
Do these really help improve the code quality that much?  Feels like an unnecessary level of indirection.  🤔

Reply:
they are probably going to be used in many places and I feel if a string or magic number is used in many places, I prefer it to be a variable

Reply:
Why do you prefer constants instead of strings?  You can do `'hello' as const` so it has an explicit type and the type checker doesn't benefit from constants here.